benefits/settings.py

```diff
@@ -234,14 +234,21 @@ def RUNTIME_ENVIRONMENT():
 STATICFILES_DIRS = [os.path.join(BASE_DIR, "benefits", "static")]
 # use Manifest Static Files Storage by default
 STORAGES = {
+    "default": {
+        "BACKEND": "django.core.files.storage.FileSystemStorage",
+    },
     "staticfiles": {
         "BACKEND": os.environ.get(
             "DJANGO_STATICFILES_STORAGE", "django.contrib.staticfiles.storage.ManifestStaticFilesStorage"
         )
-    }
+    },
 }
 STATIC_ROOT = os.path.join(BASE_DIR, "static")
 
+#  User-uploaded files
+
+MEDIA_ROOT = os.environ.get("DJANGO_DB_DIR", BASE_DIR) + "/uploads/"
```

Comment:
And circling back to @angela-tran's point from our discussion earlier this week:

It looks like `DATABASE_DIR` is a custom variable we defined, not one of the built-in Django settings. So that means it can be renamed to whatever we want -- to be more descriptive now that we're using it for more than the database.

To see which settings have been changed from the default and which are custom, run:

```console
$ python manage.py diffsettings
ALLOWED_HOSTS = ['localhost']  <--- modified from the Django default
ANALYTICS_KEY = None  ### <--- a custom setting not in the defaults
...
```

See https://docs.djangoproject.com/en/5.1/ref/django-admin/#django-admin-diffsettings